packages/devtools_app/lib/src/app.dart

```diff
@@ -46,11 +47,17 @@ const codeSizeRoute = '/codeSize';
 /// Top-level configuration for the app.
 @immutable
 class DevToolsApp extends StatefulWidget {
-  const DevToolsApp(this.screens, this.preferences, this.ideTheme);
+  const DevToolsApp(
+    this.screens,
+    this.preferences,
+    this.ideTheme,
+    this.analyticsProvider,
```

Comment:
This is going to require a paired change in google3 - we have an internal version of main.dart that initializes DevToolsApp

Reply:
Noted.